src/components/Builder/Index.jsx

```diff
@@ -82,88 +83,88 @@ export const Editor = () => {
   };
 
   return (
-    <Row
-      style={{
-        display: "flex",
-        gap: "5%",
-        justifyContent: "center"
-      }}
-    >
-      <Col
-        xs={{ span: 24 }}
-        sm={{ span: 24 }}
-        md={{ span: 12 }}
-        lg={{ span: 10 }}
-        className={styles["hide-scrollbar"]}
-        style={{
-          minheight: "98vh",
-          maxHeight: "98vh",
-          overflow: "auto",
-          padding: "10px"
-        }}
-      >
-        <Typography.Title
-          level={2}
+    <>
+      <Navbar />
+      <Row>
+        <Col
+          xs={{ span: 24 }}
+          sm={{ span: 24 }}
+          md={{ span: 12 }}
+          lg={{ span: 12 }}
+          className={styles["hide-scrollbar"]}
           style={{
-            display: "flex",
-            justifyContent: "center",
-            marginTop: "10px"
+            minheight: "98vh",
+            maxHeight: "98vh",
+            overflow: "auto",
+            padding: "2%",
```

Comment:
lets use rem instaed of percent here as well

Reply:
kk